devtools/builder/run_docker_build_conda_linux.sh

```diff
@@ -1,8 +1,8 @@
 #!/usr/bin/env bash
+set -e
 
 FEEDSTOCK_ROOT=$(cd "$(dirname "$0")/../../"; pwd;)
-# DOCKER_IMAGE=hainm/pytraj-build-box:2019-03 # centos-5; does not have python 3.7
-DOCKER_IMAGE=condaforge/linux-anvil # Miniconda only has python 3.7 on centos >= 6
+DOCKER_IMAGE=hainm/pytraj-build-box:2020-04.24  # centos-7
```

Comment:
I guess using `centos-7` image will resolve all of my previous issues. :D 
(was trying to bring `pytraj` to old distribution).